Tests/Processor/OvhLogsProcessorTest.php

```diff
@@ -28,4 +28,18 @@ public function testProcessRecordAddsTheToken()
 
         $this->assertSame(['extra' => ['X-OVH-TOKEN' => 'token']], $result);
     }
+
+    /**
+     * @covers \Chaplean\Bundle\OvhLogsBundle\Processor\OvhLogsProcessor::__construct()
+     * @covers \Chaplean\Bundle\OvhLogsBundle\Processor\OvhLogsProcessor::processRecord()
+     *
+     * @return void
+     */
+    public function testProcessRecordAddsTheEnv()
+    {
+        $processor = new OvhLogsProcessor('token', 'env');
+        $result = $processor->processRecord([]);
+
+        $this->assertSame(['extra' => ['X-OVH-TOKEN' => 'token', 'env' => 'env']], $result);
```

Comment:
We need to be able to add ANY king of extra data (like "_user", "_myownvariable") in $logger later. Why only env?